lib/project/web-site-project.ts

```diff
@@ -60,6 +61,10 @@ export class MobileWebSiteProject extends frameworkProjectBaseLib.FrameworkProje
 		throw new Error("Not applicable.");
 	}
 
+	public get relativeAppResourcesPath(): string {
+		throw new Error("Not applicable.");
```

Comment:
This is a unified method that all project-types have in our code. 
As mobile websites do not have an App_Resources folder this method (along with many others as you can see throughout this file)